i18n/de/docusaurus-plugin-content-docs/current/help-center/tim-features/contact-organizations.mdx

```diff
@@ -42,7 +43,14 @@ Sie können nun einen Chat starten. Falls Sie dabei weitere Hilfe benötigen, k
     <li>Tippen Sie auf **Organisationen** in der oberen rechten Ecke des Dialogs.</li>
     <li>Wählen Sie die Organisation, die Sie kontaktieren möchten direkt aus oder suchen Sie nach ihr.</li>
     <li>Tippen Sie auf den Kontaktpunkt, den Sie anschreiben möchten.</li>
-    <li>Tippen Sie auf **Nachricht senden**, um den Chat zu starten.</li>
+    <li>Geben Sie einen Betreff in das Eingabefeld ein.</li>
+    <li>Tippen Sie auf **Anfrage erstellen**, um den Chat zu starten.</li>
   </ol>
   </TabItem>
 </Tabs>
+
+:::info
+
+Der Betreff, den Sie eingeben, wird als Titel der Anfrage angezeigt. Falls Sie keinen Betreff eingegeben haben, wird der Titel nur den Namen des Kontaktpunktes und der Organisation, die Sie kontaktiert haben und die Web-Adresse Ihrer Organisation anzeigen.
```

Comment:
I wonder if users understand better what this implies if we write 'title of the chat'. But this is more of a thought than something that needs fixing, do with it what you please.